### DISCUSSION TITLE
macOS vibrancy support?

### DISCUSSION THREAD

**Discussion by luciascarlet** [ghostty-org/ghostty#2862] [Feature Requests, Ideas]
I wonder if it would be feasible or practical to optionally add support to use macOS's built-in "vibrancy" formulas rather than the opacity/blur combo. It definitely shouldn't replace those settings, as they for example don't allow you to use a smaller blur radius if you prefer this, but they are a noticeably different effect to alpha transparency as they also process the colours. These modes can be accessed using NSVisualEffectView.

**Comment by pluiedev** [comment]
xref #7814, #8155
